cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -1061,7 +1061,7 @@ protected String determineImagePullPolicy(ImagePullPolicy requestedImagePullPoli
             return requestedImagePullPolicy.toString();
         }
 
-        if (image.toLowerCase(Locale.ENGLISH).endsWith(":latest"))  {
+        if (image.toLowerCase(Locale.ENGLISH).contains(":latest"))  {
```

Comment:
It might be worth modifying line 1051 (the [doc comment](https://github.com/strimzi/strimzi-kafka-operator/pull/2933/files#diff-2a9350887fe59eb3c605753efd97e985R1051)) 
To read something like
```
 *    :latest* results in Always, where * can be any valid tag string, example my-image:latest or my-kafka:latest-kafka-foo.bar.baz
```

Reply:
Good catch. I fixed the Javadoc.